src/main/java/com/parkhomovsky/bookstore/dto/book/CreateBookRequestDto.java

```diff
@@ -11,22 +11,22 @@
 
 @Data
 public class CreateBookRequestDto {
-    @NotNull(message = "Title cannot be null")
+    @NotBlank(message = "Title cannot be empty")
```

Comment:
If you are checking that minimum size can be 2 it would never be empty, replace it by NotNull.